README.md

```diff
@@ -1,86 +1,126 @@
+The Docker image includes the .NET Framework 4.6.2, however, your application does not need to explicity target the .NET Framework 4.6.2. Applications that target .NET Framework 4.0 or latest should work correctly with this image.
 
-## What is the .NET Framework?
-The .NET Framework can be used to build Windows and Windows Server applications. It includes technologies such as ASP.NET Web Forms, ASP.NET MVC and Windows Communication Framework (WCF) applications. 
+## Deploying a .NET Framework 3.5 application with Docker
 
-## How to use the .NET Framework 3.5 image?
+It is easy to create a Docker image including a .NET Framework 3.5 application. You can try the instructions below or check out the [.NET Framework 3.5 Docker sample](https://github.com/Microsoft/dotnet-framework-docker-samples) if you want to try a pre-made version that's ready go.
 
-Create a Dockerfile for your application
+1. Build your application in Visual Studio or at the command line. 
+2. Add the following Dockerfile file to your project. You will need to change the `dotnetapp.exe` string to your app name. The Dockerfile assumes that your app is built to the `bin\Release` directory`. Please update your Dockerfile as appropriate. 
 
-```
+```Dockerfile
 FROM microsoft/dotnet-framework:3.5
-ADD MyDotNet35App.exe /MyDotNet35App.exe
-ENTRYPOINT MyDotNet35App.exe
+WORKDIR \app
+COPY bin\Release .
+ENTRYPOINT ["dotnetapp.exe"]
 ```
 
-You can then build and run the Docker image:
+3. Type the following Docker commands at the command line, within your project directory (beside Program.cs).
 
+```console
+docker build -t dotnetapp .
+docker run dotnetapp
 ```
-$ docker build -t my-dotnet35-app .
-$ docker run my-dotnet35-app
-```
 
-## Supported Docker Versions
+The Docker image includes the .NET Framework 3.5, however, your application does not need to explicity target the .NET Framework 3.5. Applications that target .NET Framework 1.0 through 3.5 should work correctly with this image.
+
+## Image variants
+
+The `microsoft/dotnet-framework` images come in different flavors, each designed for a specific use case.
+
+### `4.6.2`
+
+This is the defacto image. If you are unsure about what your needs are, you probably want to use this one. It is designed to be used both as a throw away container (mount your source code and start the container to start your app), as well as the base to build other images off of.
+
+This image is for .NET Framework 4.0 and later version applications. It is based on the [Windows Server Core image](https://hub.docker.com/r/microsoft/windowsservercore/).
 
-This image has been tested on Docker Versions 1.12.1-beta26 or higher.
+### `3.5`
 
-## License of this repository
+This image is for .NET Framework 3.5 and earlier version applications.  It is based on the [Windows Server Core image](https://hub.docker.com/r/microsoft/windowsservercore/).
 
-MIT
+## Related Repos and Examples
```

Comment:
I like how these are separate sections in the dotnet readme.  I also like how you had enumerated the samples and provides links to them.  I realize the framework samples are evolving.  Curious if you are planning on following the same pattern here?